jef/streetmerchant
# fix(util): proper cloudfare protection handling

### Description

Fixes #1490 

- Use a different user agent randomiser with a desktop-only filter
- This prevents issue for any Cloudfare protected webpages (such as CCL or Caseking)

### Testing

1. Search for any products on a Cloudfare-protected store (e.g. CCL)
2. Notice that it works and doesn't stay in an endless loop as before

### New dependencies

- `user-agents`
